executable/Main.hs

```diff
@@ -1 +1,14 @@
-main = putStrLn "Hello, world!"
+import qualified Fum2GitHub.Fum as Fum
+import           System.Environment (getArgs)
+
+main = do
+    args <- getArgs
+    if length args /= 2 then
```

Comment:
is better written as
```hs
case args of
  [fumApiUsersUrl, authToken] -> someotherfunc fumAPiUsersUrl authToken
  _                           -> putStrLn "Usage: fum-api-users-url authToken" >> exitWith (ExitFailure 1)
```

[System.Exit](https://hackage.haskell.org/package/base-4.7.0.2/docs/System-Exit.html)

Or one could use [optparse-applicative](https://github.com/pcapriotti/optparse-applicative) to really parse arguments (I can give small tutorial f2f).